Bottom: ea0f3bce60467d2dda331b7503567735c48224b7
Top:    1e047411c0e150a13f10660310f4449a01f93255
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 12:57:02 +0100

Refresh of convert-board-objects-to-glist

---

diff --git a/src/global.h b/src/global.h
index c162e70..9ad5331 100644
--- a/src/global.h
+++ b/src/global.h
@@ -287,7 +287,7 @@ typedef struct			/* holds information about one layer */
     TextN,			/* labels */
     PolygonN,			/* polygons */
     ArcN;			/* and arcs */
-  GList *Line;			/* pointer to additional structures */
+  GList *Line;
   GList *Text;
   GList *Polygon;
   GList *Arc;
@@ -339,8 +339,8 @@ typedef struct
   Cardinal PadN;		/* number of pads */
   Cardinal LineN;		/* number of lines */
   Cardinal ArcN;		/* number of arcs */
-  GList *Pin;			/* pin description */
-  GList *Pad;			/* pad description of SMD components */
+  GList *Pin;
+  GList *Pad;
   GList *Line;
   GList *Arc;
   BoxType VBox;
@@ -375,7 +375,7 @@ typedef struct			/* holds all objects */
     ElementN,			/* and elements */
     RatN;			/* and rat-lines */
   int LayerN;			/* number of layers in this board */
-  GList *Via;			/* pointer to object data */
+  GList *Via;
   GList *Element;
   GList *Rat;
   rtree_t *via_tree, *element_tree, *pin_tree, *pad_tree, *name_tree[3],	/* for element names */
